content/docs/user-guide/components/reference/terrain/surface-material-list.md

```diff
@@ -5,8 +5,19 @@ description: 'Open 3D Engine (O3DE) Terrain Surface Materials List component ref
 weight: 100
 ---
 
-The **Terrain Surface Materials List** component defines mappings between surface types and render materials.  You can use this to change the appearance of a surface type within different areas of your game. When you assign a material to a surface type, all the visible surfaces of that type, within the bounds of the required [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape/) component, will adopt that material.
+The **Terrain Surface Materials List** component defines mappings between surface types and render materials. You can use this to change the appearance of a surface type within different areas of your game. When you assign a material to a surface type, all the visible surfaces of that type, within the bounds of the required [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape/) component, will adopt that material.
 
+This component makes use of [macro materials](/docs/user-guide/components/reference/terrain/terrain-macro-material) and [detail materials](/docs/user-guide/components/reference/terrain/terrain-detail-material/) across a large terrain surface and blends between them. This blending of detail materials with the macro material enables you to use small, high fidelity, tiled detail materials that have variations in color and lighting across the terrain.
+
+The blending is based on the surface weights of the materials. The three surface materials with the highest weight values at every point will be blended together with relative weighting to add up to 100%. For example, say you had the following surface materials/weights:
+* Grass = 0.25
+* Dirt = 0.125
+* Sand = 0.125
+* Rock = 0.05
+
+In this scenario, the blend will ignore rock, and the result will be 50% grass, 25% dirt, and 25% sand.
```

Comment:
Nit: Preference to speak in present-tense as long as its appropriate. 
```suggestion
In this scenario, the blend ignores rock, and the result is 50% grass, 25% dirt, and 25% sand.
```

Reply:
Good call out thank you